TypeScript compile error because of incompatible override of Atomic.unpack by its subclasses

I am a TypeScript user with strict mode turned on, and in that configuration this library doesn't compile without manual modifications. I would like to address that issue for fellow strict TypeScript users, to allow them access to this otherwise great library.
### ❗  Behavior
When trying to install and thus compile the library with TypeScript, the following error occurs:
```
node_modules/osc-js/lib/osc.d.ts:200:5 - error TS2416: Property 'unpack' in type 'AtomicTimetag' is not assignable to the same property in base type 'Atomic'.
  Type '(dataView: DataView, initialOffset?: number | undefined) => number' is not assignable to type '(dataView: DataView, method: string, byteLength: number, initialOffset?: number | undefined) => number'.
    Types of parameters 'initialOffset' and 'method' are incompatible.
      Type 'string' is not assignable to type 'number'.
```
### ❔ Cause
`AtomicTimetag` is overriding its parent method `unpack` in an incompatible way. Parent method `Atomic.unpack` is defined as 
```typescript
unpack(dataView: DataView, method: string, byteLength: number, initialOffset?: number): number;
```
While `AtomicTimetag.unpack` is defined as
```typescript
unpack(dataView: DataView, initialOffset?: number): number;
```
By omitting the `method` and `byteLength` property, it renders itself incompatible with its base class, preventing TypeScript from compiling.
### ✔️ How to fix
By changing the child signature into
```typescript
unpack(dataView: DataView, initialOffset?: number|string): number;
```
it overlaps with the parent signature enough to compile. However, this is kinda hacky and the child classes should not override their base methods incompatibly.

--- a/src/atomic.js
+++ b/src/atomic.js
@@ -48,7 +48,7 @@ export default class Atomic {
    * @param {number} [initialOffset=0] Offset of DataView before unpacking
    * @return {number} Offset after unpacking
    */
-  unpack(dataView, method, byteLength, initialOffset = 0) {
+  unpackWithMethod(dataView, method, byteLength, initialOffset = 0) {
     if (!(dataView && method && byteLength)) {
       throw new Error('OSC Atomic cant\'t be unpacked without given dataView, method or byteLength')
     }

--- a/src/atomic/float32.js
+++ b/src/atomic/float32.js
@@ -33,6 +33,6 @@ export default class AtomicFloat32 extends Atomic {
    * @return {number} Offset after unpacking
    */
   unpack(dataView, initialOffset = 0) {
-    return super.unpack(dataView, 'getFloat32', 4, initialOffset)
+    return super.unpackWithMethod(dataView, 'getFloat32', 4, initialOffset)
   }
 }

--- a/src/atomic/float64.js
+++ b/src/atomic/float64.js
@@ -33,6 +33,6 @@ export default class AtomicFloat64 extends Atomic {
    * @return {number} Offset after unpacking
    */
   unpack(dataView, initialOffset = 0) {
-    return super.unpack(dataView, 'getFloat64', 8, initialOffset)
+    return super.unpackWithMethod(dataView, 'getFloat64', 8, initialOffset)
   }
 }

--- a/src/atomic/int32.js
+++ b/src/atomic/int32.js
@@ -33,6 +33,6 @@ export default class AtomicInt32 extends Atomic {
    * @return {number} Offset after unpacking
    */
   unpack(dataView, initialOffset = 0) {
-    return super.unpack(dataView, 'getInt32', 4, initialOffset)
+    return super.unpackWithMethod(dataView, 'getInt32', 4, initialOffset)
   }
 }

--- a/src/atomic/int64.js
+++ b/src/atomic/int64.js
@@ -43,6 +43,6 @@ export default class AtomicInt64 extends Atomic {
    * @return {number} Offset after unpacking
    */
   unpack(dataView, initialOffset = 0) {
-    return super.unpack(dataView, 'getBigInt64', 8, initialOffset)
+    return super.unpackWithMethod(dataView, 'getBigInt64', 8, initialOffset)
   }
 }

--- a/src/atomic/uint64.js
+++ b/src/atomic/uint64.js
@@ -42,6 +42,6 @@ export default class AtomicUInt64 extends Atomic {
    * @return {number} Offset after unpacking
    */
   unpack(dataView, initialOffset = 0) {
-    return super.unpack(dataView, 'getBigUint64', 8, initialOffset)
+    return super.unpackWithMethod(dataView, 'getBigUint64', 8, initialOffset)
   }
 }

